app/javascript/articles/components/SaveButton.jsx

```diff
@@ -33,7 +33,20 @@ export class SaveButton extends Component {
       });
     };
 
-    if (article.class_name === 'Article') {
+    const articleOwnedByCurrentUser = function (article) {
+      if (localStorage && localStorage.getItem('current_user')) {
```

Comment:
One idea that might make testing easier would be to check the `currentUser` rather than reading from localStorage. Experimenting very briefly with this suggests that writes to the localStorage in tests are accepted, but reads return null, this might be a null store.

Seeing as we also only check set and remove (not get) in the [localStoreageTest](https://github.com/forem/forem/blob/main/app/assets/javascripts/utilities/localStorageTest.js#L7) this might be very possible.